WalletWasabi.Fluent/ViewModels/AddWallet/TermsAndConditionsViewModel.cs

```diff
@@ -6,12 +7,12 @@
 
 namespace WalletWasabi.Fluent.ViewModels.AddWallet
 {
-	[NavigationMetaData(Title = "Welcome to Wasabi Wallet")]
+	[NavigationMetaData(Title = "Terms and conditions")]
 	public partial class TermsAndConditionsViewModel : RoutableViewModel
 	{
 		[AutoNotify] private bool _isAgreed;
 
-		public TermsAndConditionsViewModel(LegalChecker legalChecker, RoutableViewModel next)
+		public TermsAndConditionsViewModel(LegalChecker legalChecker, Action loginAction)
```

Comment:
This smells.

Reply:
https://media.giphy.com/media/hs6Et7TLnR4Zk3xCus/giphy.gif

Reply:
eliminated this.. shown as a dialog and awaits the result.